js/src/modals/Transfer/store.js

```diff
@@ -639,15 +624,12 @@ export default class TransferStore {
     const to = (isEth && !isWallet) ? this.recipient
       : (this.isWallet ? this.wallet.address : this.token.address);
 
-    const options = {
+    const options = this.gasStore.overrideTransaction({
```

Comment:
I didn't want to change the naming anywhere, so kept it as-is. If it bothers, can do it once there is no pressure.